ecosystem/typescript/sdk_v2/tests/unit/type_tag.test.ts

```diff
@@ -0,0 +1,291 @@
+import {
+  objectStructTag,
+  StructTag,
+  TypeTag,
+  TypeTagAddress,
+  TypeTagBool,
+  TypeTagParser,
+  TypeTagParserError,
+  TypeTagSigner,
+  TypeTagStruct,
+  TypeTagU128,
+  TypeTagU16,
+  TypeTagU256,
+  TypeTagU32,
+  TypeTagU64,
+  TypeTagU8,
+  TypeTagVector,
+} from "../../src/transactions/types/typeTag";
+import { Deserializer, Serializer } from "../../src/bcs";
+import { AccountAddress, AddressInvalidReason, ParsingError } from "../../src/core";
+
+const expectedTypeTag = {
+  string: "0x1::some_module::SomeResource",
+  address: "0x1",
```

Comment:
should we also test for other framework addresses?

Reply:
I think the only difference with the other framework addresses would be parsing on the `AccountAddress` side of things. Other than that there are already tests for making sure things like object/option/string are `0x1`

Unless I'm misunderstanding..?